spec.bs

```diff
@@ -3062,20 +3058,25 @@ A <dfn>server auction response</dfn> is a [=struct=] with the following [=struct
   : <dfn>interest group owner</dfn>
   :: The winning interest group's owner [=origin=].
   : <dfn>bidding groups</dfn>
-  :: A [=list=] of interest group owner and interest group name pairs that bid
+  :: A [=list=] of interest group [=interest group/owner=] and interest group [=interest group/name=] pairs that bid
      in this auction.
   : <dfn>score</dfn>
-  :: {{double}} or Null. TODO
+  :: {{double}} or Null. For component auctions, contains the desirability
+     for the winning bid in the component auction.
   : <dfn>bid</dfn>
-  :: {{double}} or Null. TODO.
+  :: {{double}} or Null. For component auctions, contains the modified bid
+     for the winning bid in the component auction.
   : <dfn>bid currency</dfn>
-  :: [=currency tag=] or Null. TODO.
+  :: [=currency tag=] or Null. For component auctions, contains the currency of

```

Comment:
so always null for non-component auctions? If so, let's be explicit about that. For example, https://wicg.github.io/turtledove/#generated-bid-modified-bid